.github/workflows/cmake_build.yml

```
@@ -275,7 +270,7 @@ jobs:
         git clone --recurse-submodules ${{ env.WX_GIT_URL }}
         cd wxWidgets
         git checkout ${{ env.WX_GIT_REF }}
-        cmake -S . -B cmake_build -D CMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }}  -D CMAKE_INSTALL_PREFIX=${{ env.WX_INSTALL_DIR }}
+        cmake -S . -B cmake_build -D CMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }}  -D CMAKE_INSTALL_PREFIX=${{ env.WX_INSTALL_DIR }} -D MACOSX_DEPLOYMENT_TARGET=10.12
```

```suggestion
        cmake -S . -B cmake_build -D CMAKE_BUILD_TYPE=${{ env.CMAKE_BUILD_TYPE }}  -D CMAKE_INSTALL_PREFIX=${{ env.WX_INSTALL_DIR }} -D CMAKE_OSX_DEPLOYMENT_TARGET=10.12
```
`MACOSX_DEPLOYMENT_TARGET` is the environment variable name. https://cmake.org/cmake/help/latest/envvar/MACOSX_DEPLOYMENT_TARGET.html